client/src/components/data-entry-components/DataEntryAlert.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+const DataEntryAlert = ({ alert, alertType }) => {
+
+    const [hidden, setHidden] = useState(false);
+
+    if (alert !== "") {
+        if (alertType === "error") {
+            if (!hidden) {
+                return (
+                    <div class="alert alert-danger alert-dismissible fade show" role="alert">
```

Comment:
The alert is getting hidden behind the navbar. Maybe adding an `mt-4` or `mt-5` class here would do the trick. When I tested locally `mt-4` sets up the alert to be almost flush with the navbar while `mt-5` creates some space. I'm fine with either approach.

Reply:
Fixed in latest commit.